Homework Linux answer

fbtom opened the pull request:
Provided answer for linux homework

fbtom committed:
Update linux.txt

ziobron commented:
@fbtom, we do not have your data in the Coders School database.
1. Please fill in this form https://forms.gle/rkzLsMUSBdnGQgdM9
2. Edit this PR to run this automation once again (you can just add a space in the description or title).
3. Wait one minute. If any label was added to this PR (like Standard, Premium, Podstawy C++, STL) then everything is fine :)
4. If your solution passed, but nothing happens, modify your code (eg. add an empty line) to run the workflow again.

If you don't know what to do - ask on Discord and provide a link to this PR.

fbtom committed:
Update typo

ziobron commented:
@fbtom, we do not have your data in the Coders School database.
1. Please fill in this form https://forms.gle/rkzLsMUSBdnGQgdM9
2. Edit this PR to run this automation once again (you can just add a space in the description or title).
3. Wait one minute. If any label was added to this PR (like Standard, Premium, Podstawy C++, STL) then everything is fine :)
4. If your solution passed, but nothing happens, modify your code (eg. add an empty line) to run the workflow again.

If you don't know what to do - ask on Discord and provide a link to this PR.

fbtom committed:
Apply short lines command

Apply short version of a command for wc 
Rename asterix to file name to grep only in processes.txt file

ziobron commented:
@fbtom, we do not have your data in the Coders School database.
1. Please fill in this form https://forms.gle/rkzLsMUSBdnGQgdM9
2. Edit this PR to run this automation once again (you can just add a space in the description or title).
3. Wait one minute. If any label was added to this PR (like Standard, Premium, Podstawy C++, STL) then everything is fine :)
4. If your solution passed, but nothing happens, modify your code (eg. add an empty line) to run the workflow again.

If you don't know what to do - ask on Discord and provide a link to this PR.

fbtom committed:
Correct grep call and history call 

grep should look for number lines but it previously did not.
history call number lines in short version

ziobron commented:
@fbtom, we do not have your data in the Coders School database.
1. Please fill in this form https://forms.gle/rkzLsMUSBdnGQgdM9
2. Edit this PR to run this automation once again (you can just add a space in the description or title).
3. Wait one minute. If any label was added to this PR (like Standard, Premium, Podstawy C++, STL) then everything is fine :)
4. If your solution passed, but nothing happens, modify your code (eg. add an empty line) to run the workflow again.

If you don't know what to do - ask on Discord and provide a link to this PR.

fbtom committed:
Update linux.txt

ziobron commented:
@fbtom, we do not have your data in the Coders School database.
1. Please fill in this form https://forms.gle/rkzLsMUSBdnGQgdM9
2. Edit this PR to run this automation once again (you can just add a space in the description or title).
3. Wait one minute. If any label was added to this PR (like Standard, Premium, Podstawy C++, STL) then everything is fine :)
4. If your solution passed, but nothing happens, modify your code (eg. add an empty line) to run the workflow again.

If you don't know what to do - ask on Discord and provide a link to this PR.

fbtom committed:
Update linux.txt

ziobron commented:
✅ CI Passed - Task: linux
New user detected (@fbtom).  Trying to automatically map accounts.

ziobron commented:
✅ CI Passed - Task: linux
🏅 Granted 10 XP for @fbtom
